config.yaml.sample

```diff
@@ -188,9 +188,16 @@ toi-store:
 
 # Configuration for the tiles of interest prune/garden command.
 toi-prune:
+  # location of tileserver logs
+  tile-traffic-log-path: ../tileserver/nohup.out
+
   # Connection and query configuration for a RedShift database containing
   # request information for tiles.
   redshift:
+    # if database-uri is not specified, postgresql configuration above is used
```

Comment:
I'd rather see you change the `redshift` configuration topic to something more generic so that a user can specify a generic postgres-compatible database URI rather than point off to another place.